src/express-tooling.ts

```diff
@@ -137,6 +158,9 @@ function validateScopes(req: ExtendedRequest,
   if (filteredScopes.length === 0) {
     next();
   } else {
+    if (logger) {
```

Comment:
* I think we should at least add the scopes array to the error message.
* @bzums should we force here the user into a concrete log level? If so I would more likely go with 'warn'.

Reply:
Yes, I even think `info` is more appropriate here than `error` (since this case is no "surprise" but rather one of two common cases).